shared_model/cryptography/ed25519_ursa_impl/crypto_provider.hpp

```diff
@@ -34,9 +39,10 @@ namespace shared_model {
        * @param publicKey - public key
        * @return true if verify was OK or false otherwise
        */
-      static bool verify(const Signed &signed_data,
-                         const Blob &orig,
-                         const PublicKey &public_key);
+      static bool verify(
+          const shared_model::interface::types::ByteRange &signed_data,
```

Comment:
```suggestion
          shared_model::interface::types::ByteRange signed_data,
```
https://stackoverflow.com/questions/27256377/c-view-types-pass-by-const-or-by-value
It is shown that it is better to pass view types by value because they are simple and cheap.